src/projects/projects-page.html

```diff
@@ -195,6 +207,9 @@
      _changePRStateFilter: function(e) {
        this.prStateFilter = e.detail;
        this.$.ajaxPullRequest.generateRequest();
+     },
+     _showPullRequest: function(newPullrequest) {
+       this.pullrequests = this.pullrequests.concat([newPullrequest]);
```

Comment:
The pull request will now appear at the end of the list, which may not be intuitive. Maybe we should extract an issue to the backlog for this.